File: Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/elijah.cs
public class Solution
{
    public IList<string> FizzBuzz(int n)
    {
        List<string> result = new();
        
        for (int i = 1; i < n + 1; i++)
        {
            if ((i % 3 == 0) && (i % 5 == 0))
            {
                result.Add("FizzBuzz");
            }
            else if (i % 3 == 0)
            {
                result.Add("Fizz");
            }
            else if (i % 5 == 0)
            {
                result.Add("Buzz");
            }
            else
            {
                result.Add(i.ToString());
            }
        }
        
        return result;
    }
}

File: Week 1/Weekly Practice Problems (Week 1)/2413: Smallest Even Multiple/elijah.cpp
class Solution {
public:
    int smallestEvenMultiple(int n)
    {
        for (int i = 1; true; i++)
        {
            if (i % 2 == 0 && i % n == 0)
            {
                return i;
            }
        }
    }
};
